common/common.go

```diff
@@ -180,6 +182,13 @@ func WithNetGRPCOptions(opts ...grpc.ServerOption) NetOption {
 	}
 }
 
+func WithPubSubDisabled() NetOption {
```

Comment:
How about `WithNetPubsub`, and when setting up `NetConfig`, use set `PubSub` to true by default?

Reply:
Opps, I forgot `NetConfig` is public. In any case, given your findings, I'm fine with letting pubsub be _off_ by default. So, same comment as before around semantics, but no need to have `PubSub` true by default.

Reply:
(We can do a minor version bump)

Reply:
Sure, now it looks more consistent.